src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,53 @@
+package racingcar.domain;
+
+import java.util.Objects;
+import racingcar.dto.CarState;
+
+public class Car implements Comparable<Car>{
+
+    private int position;
+
+    private final String name;
+    public Car(String name) {
+		this(name, 0);
+	}
+
```

Comment:
보통 팀에서 코드는 혼자 작성하지 않을 것 같은데요. 그래서 가독성이 굉장히 중요합니다. 
그래서 컨벤션이 존재하고요. 개행 띄어쓰기에 대한 컨벤션도 존재해요. 이를 지키며 코딩해 보면 좋을 것 같아요. 

```suggestion
public class Car implements Comparable<Car> {

    private int position;
    private final String name;
    
    public Car(String name) {
        this(name, 0);
    }

```